audit.go

```diff
@@ -0,0 +1,60 @@
+package authaus
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/IMQS/serviceauth"
+)
+
+type Auditor interface {
+	AuditUserAction(identity, password, actionDescription string)
+}
+
+type Action struct {
+	Who     string         `json:"who"`
+	DidWhat string         `json:"did_what"`
+	ToWhat  string         `json:"to_what"`
+	AtTime  int64          `json:"at_time"`
+	Context *ActionContext `json:"context"`
+}
+
+type ActionContext struct {
+	Location string `json:"location"`
+}
+
+func (x *Central) AuditUserAction(identity, password, actionDescription string) {
+	action := &Action{}
+	action.Who = identity
+	action.DidWhat = actionDescription
+	action.ToWhat = "IMQS Web V8"
+	action.AtTime = time.Now().Unix()
+
+	actionContext := &ActionContext{}
+	actionContext.Location = time.Now().Location().String()
```

Comment:
I don't think that adding the server's time zone is useful information in the logs. Rather just omit information that we don't have.